Quadriga/src/main/java/edu/asu/spring/quadriga/dao/workbench/impl/RetrieveProjectDAO.java

```diff
@@ -91,6 +91,9 @@
                             "Select projWork.projectDTO from ProjectWorkspaceDTO projWork where projWork.workspaceDTO in (Select wcDTO.workspaceDTO from WorkspaceCollaboratorDTO wcDTO where wcDTO.collaboratorDTOPK.collaboratoruser = :collaboratoruser)");
             query.setParameter("collaboratoruser", sUserName);
             projectDTOList = query.list();
+            for(ProjectDTO projectDTO :projectDTOList){
+                System.out.println("Project ID: "+projectDTO.getId()+" , Project Name: "+projectDTO.getProjectname()+" , "+projectDTO.getProjectWorkspaceDTOList().size());
+            }
```

Comment:
remove!!!! please don't use sysout (unless you remove right after you've check what you wanted to check).